### ISSUE TITLE
P40 pro - unable to downgrade from Emui 11 to 10

### ISSUE THREAD

**Issue by mmgi77** [ProfessorJTJ/HISuite-Proxy#176]
Hello,

I've been trying to downgrade a P40 Pro (ELS-NX9) from Emui 11.0.0.260 to 10, without success. The Hisuite proxy won't patch Hisuite v11.0.0.500, it returns a "Patch failed. Aborting process" error. 

Hisuite 10 cannot be installed on the device, as it continuously requests location access, which cannot be granted. Having Hisuite 10 on the computer and Hisuite 11 on the device doesn't work. 

The device has been downgraded from Emui 12 to 11 with Hisuite. But it won't go lower than this version, there is no other downgrade option available in Hisuite 10 or 11.

Are there any working solutions in this case?

**Comment by ProfessorJTJ** [comment]
Hi, use hisuite 11.0.0.530 or 510

https://github.com/ProfessorJTJ/HISuite-Proxy/releases/download/2.3.6/HiSuite_11.0.0.530_OVE.rar

**Comment by mmgi77** [comment]
Thank you for the quick reply!

The patch worked with this version, indeed. However, now the Hisuite reports "no internet connection" after patching. The proxy settings are correct in the Hisuite app, and both apps (hisuite and hisuite proxy) are allowed in the firewall. The proxy has been run as administrator.

What should I do next?

**Comment by ProfessorJTJ** [comment]
> Thank you for the quick reply!
> 
> The patch worked with this version, indeed. However, now the Hisuite reports "no internet connection" after patching. The proxy settings are correct in the Hisuite app, and both apps (hisuite and hisuite proxy) are allowed in the firewall. The proxy has been run as administrator.
> 
> What should I do next?

You need to open a hisuite with red icon and name HiSuite 11.exe, it will be on desktop, don't open the green hisuite.

Here's a detailed guide

https://www.youtube.com/watch?v=CdYpZlXlgOQ

**Comment by mmgi77** [comment]
Thank you again for the help. I got confused between several videos showing the process, and I missed the part about the new launch link.

The proxy works now, and I made several attempts at downgrading the firmware version (from 11.0.0.260 to 10.0.0.114, 121, 131, 176), all unsuccessful. The firmware downloads and is copied on the phone, but when the phone restarts, there is always an error. The revision is correct (R5) and the CUST/PRELOAD versions appear to be the right ones. This doesn't seem to be an issue related to the HiSuite proxy, so this case can be closed.
